QRCoder/QRCodeGenerator.cs

```diff
@@ -866,18 +874,13 @@ private static BitArray PlainTextToBinaryAlphanumeric(string plainText)
             return codeText;
         }
 
-        /// <summary>
-        /// Returns a string that contains the original string, with characters that cannot be encoded by a
-        /// specified encoding (default of ISO-8859-2) with a replacement character.
-        /// </summary>
-        private static string ConvertToIso8859(string value, string Iso = "ISO-8859-2")
-        {
-            Encoding iso = Encoding.GetEncoding(Iso);
-            Encoding utf8 = Encoding.UTF8;
-            byte[] utfBytes = utf8.GetBytes(value);
-            byte[] isoBytes = Encoding.Convert(utf8, iso, utfBytes);
-            return iso.GetString(isoBytes);
-        }
+        private static readonly Encoding _iso8859_1 =
+#if NET5_0_OR_GREATER
+            Encoding.Latin1;
+#else
+            Encoding.GetEncoding(28591); // ISO-8859-1
+#endif
+        private static Encoding _iso8859_2;
```

Comment:
On .NET 5 and newer there is a static property available for ISO-8859-1.  Regardless, for all platforms, ISO-8859-1 is guaranteed to be available.  However, requesting ISO-8859-2 will throw an exception on Linux unless the code page is installed, so it is lazy loaded and cached.  This represents no change to behavior, as attempting to use ISO-8859-2 with v1.5.1 on Linux currently throws an exception (unless the code page provider is installed).